File: level 20/classwork/cw.py
def greet():#defit vadzlevt funqcias saxels da tviton funqcia aris hello luka
    print('hello luka')
greet()
def add():
    print(5+4)
add()    

def add(num1,num2,operation):
    result=0
    if operation == "+":
        result=num1+num2
    elif operation =="-":
        result=num1-num2
    elif operation =="*":
        result =num1*num2    
    elif operation =="/":
        result=num1/num2
    else:
        result='not number'
    return result    
    
print(add(3,5,'+'))
    

File: level 064/classwork/cw.js
function checkNumbers(num1, num2) {
    // შემოწმება: პირველი რიცხვი კენტია და მეტია მეორეზე, მეორე კი ლუწია
    if (num1 % 2 !== 0 && num1 > num2 && num2 % 2 === 0) {
      return true;
    }
    return false;
  }
  